client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -82,11 +82,16 @@ export function* createSite(
 		options: {
 			site_vertical: siteVertical?.id,
 			site_vertical_name: siteVertical?.label,
+			// untranslated vertical slug
+			// so we can match directories in
+			// https://github.com/Automattic/wp-calypso/tree/master/static/page-templates/verticals
+			// TODO: determine default vertical should user input match no official vertical
+			site_vertical_slug: siteVertical?.slug || 'football',
```

Comment:
fwiw in `/start` we weren't sending a `site_vertical_slug` param; `site_vertical` is an ID so already acted as an untranslated identifier.

Maybe the folder names in `page-templates/verticals` should vertical ids rather than labels 🤔 

Anyway, my vote would be to leave out this param if the user typed a customer vertical. Headstart falls back to some default in this case ("p1" or something like that), and maybe the the template mill should be doing the same if it can't find a match.

Reply:
> Anyway, my vote would be to leave out this param if the user typed a customer vertical.

Good idea. It's probably the least we can do for now. I'll create an issue to change the `page-templates/verticals` folders to vertical IDs.

Reply:
Heh, found this from the other end p1585743383271800-slack-gutenboarding

Reply:
Now you know how deep the 🐰hole goes... 🕶